### ISSUE TITLE
Overfitting Issue in PNDM to SDv1 transfer.

### ISSUE THREAD

**Issue by freshjh** [ZhendongWang6/DIRE#38]
I trained the model on lsun_bedroom-PNDM with the default config, and test the trained model in PNDM, LDM, ADM and SDv1.

The Acc is,
PNDM: 1.00
LDM:0.99
ADM:0.99
SDv1:0.0005.

I thought the model is overfitted to the pattern of training data(PNDM).
Has anyone encountered this problem?

Thanks a lot.

**Comment by freshjh** [comment]
I solved this problem by renaming dataset folders. 
Before changes: 
-  train: /path/train/lsun_bedroom/real         /path/train/lsun_bedroom/pndm            (pndm-labeled-0,    real-labeled-1）
-  test: /path/test/lsun_bedroom/real            /path/test/lsun_bedroom/sdv1               (real-labeled-0,       sdv1-labeled-1)

After changes:
-  train: /path/train/lsun_bedroom/0_real         /path/train/lsun_bedroom/1_fake        (0_real-labeled-0,       1_fake-labeled-1)
-  test: /path/test/lsun_bedroom/0_real            /path/test/lsun_bedroom/1_fake         (0_real-labeled-0,       1_fake-labeled-1)

This is casued by the label assign mechanism of torchvision.dataset.imagefolder.

Hope this can be benificial to someone who encounter the same problem.
